config/packages/framework.yaml

```diff
@@ -15,3 +15,6 @@ framework:
     #fragments: true
     php_errors:
         log: true
+
+    # Enables annotations validation
+    validation: { enable_annotations: true }
```

Comment:
I add this line to enable validation, because i read that in the doc.  
[https://symfony.com/doc/current/validation.html#configuration](https://symfony.com/doc/current/validation.html#configuration)

Reply:
This config is done automatically in the `validator.yaml` file instantiate by flex